cln/cln_client.go

```diff
@@ -86,11 +86,6 @@ func (c *ClnClient) OpenChannel(req *lightning.OpenChannelRequest) (*wire.OutPoi
 		m := uint16(*req.MinConfs)
 		minConfs = &m
 	}
-	var minDepth *uint16
-	if req.IsZeroConf {
-		var d uint16 = 0
-		minDepth = &d
-	}
```

Comment:
Removing this is the reason why you get `CHANNELD_AWAITING_LOCKIN`. The `minDepth` parameter is used to determine the number of confirmations before we consider the channel active. That should be zero if it's a zero conf channel.